digitaltwin_client/tests/dt_e2e/service/dt_e2e_test.ts

```diff
@@ -311,6 +312,58 @@ function veryAllInterfacesAreRegistered(done) {
     })
 }
 
+//
+// veryAllInterfacesAreRegistered makes sure that the e2e test interfaces were successfully registered.  These are stored
+// in properties of the modelDiscovery interface
+//
+function runTestCommandAndCheckMessageAtEventHub(testData:CommandTestData, numberInvocationAttempts:number, done) {
+    let receiveOK = false;
+    let ehClient: EventHubClient;
+
+    const startAfterTime = new Date(Date.now());
+    const onEventHubError = (err) => {
+      throw err;
+    };
+    const onEventHubMessage = (receivedMsg) => {
+        if((receivedMsg.annotations['iothub-connection-device-id'] === testDeviceInfo.deviceId) &&
+        (receivedMsg.annotations['iothub-interface-name'] === testData.interfaceName)){
```

Comment:
>receivedMsg [](start = 9, length = 11)

Could we also please check for a PnP specific annotation here?  I think in current SDK version it's like iothub-interface-instancename (that'll change at some point).  See DT_InterfaceClient_CreateTelemetryMessage() setting DT_INTERFACE_NAME_PROPERTY for what device code specifically I'd like to test is correctly applying this annotation in e2e fashion.  Device Explorer or else Miguel/Sam can help with the exact name of this thing as it comes to EventHub.

Reply:
Ugh, never mind.  You're doing this already and I missed.  Resolving.

---
In reply to: [354605504](https://github.com/Azure/azure-iot-sdk-c/pull/1361#discussion_r354605504) [](ancestors = 354605504)